### DISCUSSION TITLE
SMB Administrative shares (c$)

### DISCUSSION THREAD

**Discussion by hme3** [iterate-ch/cyberduck#15949] [Q&A]
Is there a way to access administrative shares (c$, d$...) using SMB

**Comment by AliveDevil** [comment]
I do see the administrative shares on a Linux Samba box.

![image](https://github.com/iterate-ch/cyberduck/assets/734738/5e8e1355-f2f8-44bc-a256-97d708301abf)

If you don't see these, you can try to show hidden files (View -> Show Hidden).
If that doesn't work you can also try to use `Go -> Go To Folder` to directly navigate to your shares (e.g. enter "c$" into the path field).
